otel/otel.go

```diff
@@ -0,0 +1,58 @@
+package otel
+
+import (
+	"context"
+	"time"
+
+	"go.temporal.io/sdk/activity"
+	"go.temporal.io/sdk/workflow"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+
+	"go.opentelemetry.io/otel/trace"
```

Comment:
```suggestion
	"go.temporal.io/sdk/activity"
	"go.temporal.io/sdk/workflow"
	"go.opentelemetry.io/otel"
	"go.opentelemetry.io/otel/attribute"
	"go.opentelemetry.io/otel/trace"
```

No consistency with the separation of packages here